services/broker/start-rabbit.sh

```diff
@@ -11,18 +11,19 @@ if [ -e /etc/rabbitmq/definitions.json ]; then
 	/bin/ep /etc/rabbitmq/definitions.json
 fi
 
+# Replace ENV values in definitions.json
+if [ -e /etc/rabbitmq/rabbitmq.conf ]; then
+	/bin/ep /etc/rabbitmq/rabbitmq.conf
+fi
+
 # Check if the container runs in Kubernetes/OpenShift
 if [ -z "$POD_NAMESPACE" ]; then
   # Single container runs in docker
-  echo "POD_NAMESPACE not set, spin up single node"
-  exec docker-entrypoint.sh rabbitmq-server
+  echo "running in Docker, spin up single node"
+else
+  # Single container runs in kubernetes
+  echo "running in Kubernetes, spin up single node"
 fi
```

Comment:
is this even required? nothing changes whether it is in docker or kubernetes here

Reply:
no, not required, but whilst we find a way to start rabbit generically, it can stay